docs/content/en/docs/packages/packagecontroller.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "Manging the package controller"
+linkTitle: "Package controller"
+weight: 4
+aliases:
+    /docs/tasks/packages/packagecontroller/
+---
+
+### Install package controller after installation
+
+{{% alert title="Important" color="warning" %}}
+**Recommendation**: Ideally, you should never delete package controller after installation. When the package bundle controller resource is deleted and then gets recreated, the active bundle recorded in the package bundle controller resource will point to the latest bundle, but you can always go back to the previous bundle version. See more information about managing package bundles [here]({{< relref "./packagebundles" >}})
```

Comment:
it's we are not supposed to delete package controller, or package bundle controller CR? Which one will trigger the active bundle to point to latest?

Reply:
I'll move this down to the section about manually upgrading the package controller, where this is more relevant